src/Common/test/Common.Expression.Test/Spring/SpelDocumentationTests.cs

```diff
@@ -14,7 +14,7 @@ namespace Steeltoe.Common.Expression.Test.Spring;
 
 public sealed class SpelDocumentationTests : AbstractExpressionTests
 {
-    private static readonly Inventor Tesla = new("Nikola Tesla", new DateTime(1856, 7, 9), "Serbian")
+    private static readonly Inventor Tesla = new("Nikola Tesla", new DateTime(1856, 7, 9, 0, 0, 0, DateTimeKind.Local), "Serbian")
```

Comment:
I think this rule is unnecessary noise in tests. I see the merit of it in general, but a test is highly likely to be a source of timekind error.